src/styles/1-helpers/_mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin flex-centering() {
+  display: flex;
+  justify-content: space-between;
```

Comment:
Why is your mixin called `flex-centering` if you do `justify-content: space-between`? 
It would be better then to call it some type of `flex-alignment`.